docs/tutorials/08_quantum_kernel_trainer.ipynb

```diff
@@ -286,12 +1293,12 @@
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "{   'optimal_parameters': {ParameterVectorElement(θ[0]): 1.2984045829959046},\n",
-      "    'optimal_point': array([1.29840458]),\n",
-      "    'optimal_value': 11.733763373742224,\n",
+      "{   'optimal_parameters': {ParameterVectorElement(θ[0]): 1.6595658861350273},\n",
+      "    'optimal_point': array([1.65956589]),\n",
+      "    'optimal_value': 12.928464717873807,\n",
```

Comment:
These values seem quite different from before - as well as the accuracy below which as gone from 1.0 to 0.5. If you run this does it change each time? The changes that you did, simple renames, should of course not affect this. However I recall SPSA having some random aspects so maybe the algorithm_globals seed should be set to allow the results here to be reproducible as having the results vary each time can be unhelpful for end-users unless that is truly the intent.